### PR TITLE
Feat: Adding ssh ProxyJump to ssh_master template

### PR DESCRIPTION
**Pull request by oxedions**
Adding icebergs ssh ProxyJump capability in ssh_master
Adding a defaults value to disable by default ssh jump

### CODE REVIEW

File: roles/core/ssh_master/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+icebergs_system_enable_ssh_jump: false
```
[the remark this answers is not in the document]
Comment (oxedions) [reply roles/core/ssh_master/defaults/main.yml:1]: Yes, you are right. Forgot 🙂

File: roles/core/ssh_master/readme.rst [hunk 2]
```diff
@@ -93,6 +96,38 @@ first one to break in case of uncoherent inventory. If this happens, check your
...
+iceberg_ssh_jump_target:
+
+.. code-block:: text
+
+  [iceberg3:vars]
+  iceberg_master = iceberg1
+  iceberg_level = 2
+  iceberg_ssh_jump_target = 10.10.0.77
+
+In case of issue, try adding verbosity to the ss invocation to investigate (-vvv).
```
[the remark this answers is not in the document]
Comment (oxedions) [reply roles/core/ssh_master/readme.rst:129]: Good catch !

**Comment by oxedions** [comment]
Missing changelog

**Review by btravouillon — CHANGES_REQUESTED** [review]
There is an error in the readme.

File: roles/core/ssh_master/readme.rst [hunk 3]
```diff
@@ -101,26 +136,26 @@ Mandatory inventory vars:
...
 
 * network_interfaces[item]
+* icebergs_system
 
 Optional inventory vars:
 
 **hostvars[inventory_hostname]**
 
 * security.ssh.hostkey_checking
+* icebergs_system_enable_ssh_jump
```
[the remark this answers is not in the document]
Comment (oxedions) [reply roles/core/ssh_master/readme.rst]: Please have a look on last commit. I updated 2 variables names, to comply with standards.

**Review by btravouillon — DISMISSED** [review]
Looks good to me. 👍

**Pull request merged**